[seq66] unable to build

I have tried to build this a few times and i always run into errors. I am running arch linux and so I use yay.  I would expect all of the dependencies to be included so I am not sure what the error is about but I am getting the following on build

``nsm/nsmbase.cpp: In member function 'void seq66::nsmbase::add_server_method(seq66::nsm::tag, lo_method_handler)':
nsm/nsmbase.cpp:885:37: error: cannot convert 'lo_server_thread' {aka 'lo_server_thread_*'} to 'lo_server' {aka 'lo_server_*'}
  885 |         (void) lo_server_add_method(m_lo_server_thread, m, p, h, this);
      |                                     ^~~~~~~~~~~~~~~~~~
      |                                     |
      |                                     lo_server_thread {aka lo_server_thread_*}
In file included from /usr/include/lo/lo.h:32,
                 from ../include/nsm/nsmbase.hpp:28,
                 from nsm/nsmbase.cpp:126:
``

--- a/packages/seq66/.gitignore
+++ b/packages/seq66/.gitignore
@@ -1,0 +1,1 @@
+/fix-liblo.patch

--- a/packages/seq66/PKGBUILD
+++ b/packages/seq66/PKGBUILD
@@ -3,22 +3,26 @@
 
 pkgname=seq66
 pkgver=0.99.12
-pkgrel=1
+pkgrel=2
 pkgdesc='A live-looping MIDI sequencer with a Qt graphical interface'
 arch=(aarch64 x86_64)
 url='https://github.com/ahlstromcj/seq66'
-license=(GPL2)
+license=(GPL-2.0-only GPL-3.0-or-later)
 depends=(gcc-libs glibc graphite qt5-base)
 makedepends=(alsa-lib git jack liblo)
 groups=(pro-audio)
 source=("$pkgname-$pkgver.tar.gz::https://github.com/ahlstromcj/$pkgname/archive/refs/tags/$pkgver.tar.gz"
+        'fix-liblo.patch::https://github.com/ahlstromcj/seq66/commit/666a467b.patch'
         'seq66-docdir.patch')
 sha256sums=('c840cf85f463a4888ed9454c79fe22e812dece70c1e6b78970b5330573f3f332'
+            'ca820cc9763d7217d2782a51bf2d4a072c132dd4065153a9b81a717161bce0cc'
             'c689e2bfc95002483830c4ddc75694748be6f40a6a88ca8f983640f558285558')
 
 prepare() {
   cd $pkgname-$pkgver
   patch -p0 -N -r - -i "$srcdir"/seq66-docdir.patch
+  # fix liblo 0.32 header compatibility
+  patch -Np1 -i "$srcdir"/fix-liblo.patch
 }
 
 build() {

